extensions/BugModal/web/autosize.min.js

```diff
@@ -1,6 +1,6 @@
 /*!
-	Autosize 3.0.20
+	Autosize 4.0.0
```

Comment:
we're forking the autosize library due to extensive changes, so this is not so important. The version number is updated and it comes from the same github org, so that's good enough for me.